infra/website/docs/blog/retrieval-augmentation-with-feast.md

```diff
@@ -0,0 +1,167 @@
+question and answering).
+
+The typical RAG process involves:
+1. Sourcing text data relevant for your application
+2. Transforming each text document into smaller chunks of text
+3. Transforming those chunks of text into embeddings
+4. Inserting those chunks of text along with some identifier for the chunk and document in some database
+5. Retrieving those chunks of text along with the identifiers at run-time to inject that text into the LLM's context
+6. Calling some API to run inference with your LLM to generate contextually relevant output
+7. Returning the output to some end user
+
+Implicit in (1)-(4) is the potential of scaling to large amounts of data (i.e., using some form of distributed computing),
+orchestrating that scaling through some batch or streaming pipeline, and customization of key transformation decisions
+(e.g., tokenization, model, chunking, data format, etc.).
+
+# What is the scope of Retrieval?
+So, the Retrieval in RAG also includes:
+1. Ingestion
+1. Transformation
+1. Indexing
+1. Retrieval
+
+RAG patterns often use vector similarity search for the retrieval step, but it is worth emphasizing that this is not the
+only retrieval pattern that can be useful. In fact, standard entity-based retrieval can be very powerful for 
+applications where relevant user-context is necessary.
+
+# How does that relate to Feast?
+Feast is focused on supporting the lifecycle of data for AI/ML applications. Feast was built to support the
+offline training, online serving, and metadata management so that users can successfully scale their production AI
+applications.
+
+What that means is that Feast can help you not only serve your documents, user data, and other metadata for production 
+RAG applications, but it can also help you scale your embeddings on large amounts of data (e.g,. using Spark to embed 
+gigabytes of documents), re-use the same code online and offline, prepare your datasets so you can fine tune your
+embedding, generative, or retrieval models later, and track changes to your transformations, data sources, and 
+RAG-sources to provide you with replayability and data lineage.
+
+Historically, Feast catered to Data Scientists and ML Engineers who implemented their own types of data/feature transformations but, now, 
+many RAG providers handle this out of the box for you. We will invest in creating extendable implementations to make it easier 
+to customize your applications.
+
+# How might I want to customize retrieval?
+As mentioned, Feast allows you to customize:
+- Chunking
+- Tokenization
+- Embedding
+- The offline engine to embed huge amounts of documents
+- The metadata you may want to structure or weight differently during retrieval
+- Hybrid Retrieval strategies combining dense and sparse retrieval
+- Reranking mechanisms
+- Fine Tuning embedding, retrieval, and generative models
+
+# Feast Powered by Milvus
```

Comment:
I'd spell out what the user should be getting. "Here's an example of using Feast to do X Y and Z. Milvus provides B, while Feast makes it easier to do A"